purchase_procurement_analytic/models/stock.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, models
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    @api.model
+    def _prepare_procurement_from_move(self, move):
+        res = super(StockMove, self)._prepare_procurement_from_move(move)
+        res['account_analytic_id'] = move.procurement_id.account_analytic_id.id
```

Comment:
Yes. 
If account_analytic_id in stock.move is needed I can create other module taking the value from procurement.